bendigobank.rb

```diff
@@ -0,0 +1,87 @@
+require 'watir'
+require 'json'
+require 'nokogiri'
+require 'date'
+require_relative 'account'
+require_relative 'transaction'
+
+class Bendigobank
+
+  def self.execute
+    connect
+    fetch_accounts(connect)
+    fetch_transactions(connect)
+  end
+
+  def self.connect
+    Watir.logger.ignore :deprecations
+    browser = Watir::Browser.new :firefox
+    browser.goto 'http://demo.bendigobank.com.au'
+    browser.button(name: 'customer_type').click
+    return browser
```

Comment:
No need `return` at the end of the method. Ruby return value of the last line of the method.